src/plugins/twitter/twitter.js

```diff
@@ -40,37 +41,93 @@ var componentName = "wb-twitter",
+				// -Timeline widgets stopped honouring tweet limits on July 21, 2023 and began showing up to 100 tweets at a time ("verified" accounts only)
+				// -Facebook page plugin's default height is documented in https://developers.facebook.com/docs/plugins/page-plugin#settings
+				if ( twitterLink.dataset.height === "fb-page" || ( twitterLink.dataset.tweetLimit && !twitterLink.dataset.height ) ) {
+					twitterLink.dataset.height = "500";
+				}
+
+				// Add a "do not track" parameter (i.e. data-dnt="true" attribute) unless it's already been set
+				// Note: Covered in https://developer.twitter.com/en/docs/twitter-for-websites/webpage-properties
+				if ( !twitterLink.dataset.dnt ) {
+					twitterLink.dataset.dnt = "true";
+				}
+
 				// Add a loading icon below the link
 				loadingDiv.className = "twitter-timeline-loading";
 				twitterLink.after( loadingDiv );
 
-				// Remove the loading icon after the timeline widget appears
-				// Note: Twitter's widget script removes "a.twitter-timeline" upon filling-in the timeline's content... at which point the loading icon is no longer useful
+				// Observe DOM mutations
 				observer = new MutationObserver( function( mutations ) {
-
-					// Check for DOM mutations
 					mutations.forEach( function( mutation ) {
+						switch ( mutation.type ) {
 
-						// Deal only with removed HTML nodes
-						mutation.removedNodes.forEach( function( removedNode ) {
+						// Check for attribute changes
+						case "attributes": {
+							const mutationTarget = mutation.target;
 
-							// If the removed node was a Twitter link, remove its adjacent loading icon and stop observing
-							if ( removedNode === twitterLink && mutation.nextSibling === loadingDiv ) {
-								loadingDiv.remove();
-								observer.disconnect();
+							// Override the timeline iframe's title right after Twitter's widget script adds it
+							// Note: The timeline's iframe title is English-only and written in title case ("Twitter Timeline")... This replaces it with an i18n version written in sentence case.
+							if ( mutationTarget.nodeName === "IFRAME" && mutationTarget.title !== i18nText.timelineTitle ) {
+								mutationTarget.title = i18nText.timelineTitle;
 							}
-						} );
+							break;
+						}
+
+						// Check for node removals
+						case "childList": {
+							mutation.removedNodes.forEach( function( removedNode ) {
+
+								// If the removed node was a Twitter link, remove its adjacent loading icon, add skip links and stop observing
+								// Note: Twitter's widget script removes "a.twitter-timeline" upon displaying the timeline iframe's content... at which point the loading icon is no longer useful
+								if ( removedNode === twitterLink && mutation.nextSibling === loadingDiv ) {
+									const iframeContainer = loadingDiv.previousElementSibling;
+
+									loadingDiv.remove();
+									addSkipLinks( iframeContainer );
+									observer.disconnect();
```

Comment:
Add the following lines to force Safari to display the iframe when it is updated:

```suggestion
									

// The following 2 lines were added as a workaround in Safari where the iFrame is not displayed
$( ".wb-twitter" ).css( "opacity", 1 );
$( ".wb-twitter" ).css( "opacity", "" );
									
observer.disconnect();
```